srcpkgs/povray/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'povray'
+pkgname=povray
+version=3.7.0.7
+revision=1
+_v=${version%.*.*}
+hostmakedepends="boost-devel git glu-devel autoconf automake zlib-devel libpng-devel libjpeg-turbo-devel tiff-devel SDL-devel"
```

Comment:
`*-devel` packages should probably be in makedepends. 
Also don't you want `SDL2-devel` instead of `SDL-devel`?

Reply:
No I'm sure, I gave SDL2 a try first, didn't work (namely it wasn't even detected as SDL).

Reply:
Then you have to fix you depends since you have SDL2 in there.